packages/block-library/src/media-text/media-container.js

```diff
@@ -81,6 +92,8 @@ class MediaContainer extends Component {
 				onSelect={ onSelectMedia }
 				accept="image/*,video/*"
 				allowedTypes={ ALLOWED_MEDIA_TYPES }
+				notices={ noticeUI }
```

Comment:
`noticeUI` is not fitting properly here, should be addressed in a separate issue:
![image](https://user-images.githubusercontent.com/1477453/55280228-27539900-532b-11e9-8d70-dabfe4431905.png)